lib/main.dart

```diff
@@ -3,6 +3,8 @@ import 'package:flutter/material.dart';
 import 'package:standup_randomizer/pages/pages.dart';
 import 'package:standup_randomizer/widgets/widgets.dart';
 
+import 'widgets/circular_list.dart';
```

Comment:
Please export `circular_list.dart` it as part of the widgets.dart file which is already included above

Reply:
i'll merge the change and add a follow up commit :)